sqlglot/dialects/oracle.py

```diff
@@ -166,6 +168,7 @@ class Tokenizer(tokens.Tokenizer):
             **tokens.Tokenizer.KEYWORDS,
             "(+)": TokenType.JOIN_MARKER,
             "COLUMNS": TokenType.COLUMN,
+            "KEEP": TokenType.KEEP,
```

Comment:
you only need one or the other, since you specify window_before_paren_token, then everyone can parse keep as a token and then you can put this into id_var, or you can get rid of line 64,

Reply:
I think I made the change you suggested, but let me know if I missed something.